backend/app/api/v1/endpoints/ilab/ilab.py

```diff
@@ -300,23 +300,34 @@ async def run_samples(
         200: example_response(
             [
                 {
-                    "begin": "2024-09-12 17:40:27.982000+00:00",
-                    "end": "2024-09-12 18:03:23.132000+00:00",
-                    "id": "6BA57EF2-7139-11EF-A80B-E5037504B9B1",
+                    "begin": "2024-10-29 14:30:56.723000+00:00",
```

Comment:
Consider moving the example responses out of the API function and into the shared location `commons/example_responses.py`

Reply:
I've gone back and forth on that. On one hand, it's useful documentation alongside the declaration. On the other hand, it's a large body of text that can be disruptive in-line. I suspect that when I originally wrote the ILAB API I wasn't aware of that convention. Now I hesitate to move it because it'll cause more merge collisions further along in the chain. I'd rather save move/refactoring stuff until everything's landed on `main`. This cherry-pick rebase procedure has been painful enough without adding anything that's not strictly necessary along the way!